salt/modules/mac_service.py

```diff
@@ -113,6 +113,45 @@ def _always_running_service(name):
     return False
 
 
+def _get_domain_target(name, service_target=False):
+    '''
+    Returns the domain/service target and path for a service. This is used to
+    determine whether or not a service should be loaded in a user space or
+    system space.
+
+    :param str name: Service label, file name, or full path
+
+    :param bool service_target: Whether to return a full
+    service target. This is needed for the enable and disable
+    subcommands of /bin/launchctl. Defaults to False
+
+    :return: Tuple of the domain/service target and the path to the service.
+
+    :rtype: tuple
+    '''
+
+    # Get service information
+    service = _get_service(name)
+
+    # get the path to the service
+    path = service['file_path']
+
+    # most of the time we'll be at the system level.
+    domain_target = 'system'
+
+    # check if a LaunchAgent as we should treat these differently.
+    if 'LaunchAgents' in path:
+        # Get the console user so we can service in the correct session
+        uid = salt.utils.mac_utils.console_user()
```

Comment:
can you switch these to using `__utils__['mac_utils.console_user']()` instead of using the import `salt.utils.mac_utils`?

It allows for people that use the dropin dynamic modules to actually replace the modules that use newer salt utils.

Thanks,
Daniel

Reply:
@gtmanfred would like me to convert the other mac_util calls as well?

Reply:
Eventually we should go through and change them all to using `__utils__`  i was just think about these ones since you are adding new ones.

Reply:
Sounds good, I changed all my additions to using `__utils__`

Reply:
I'm still seeing `salt.utils.mac_utils.console_user()` need change it to `__utils__['mac_utils.console_user']`